zaza/openstack/charm_tests/ceph/iscsi/tests.py

```diff
@@ -199,7 +199,13 @@ async def check_device_present():
             run = await zaza.model.async_run_on_unit(
                 ctxt['client_entity_id'],
                 ls_bdevice_cmd.format(bdevice=device_ctxt['bdevice']))
-            return device_ctxt['bdevice'] in run['Stdout']
+            # Juju have moved to using 'stdout' rather than 'Stdout' so handle
+            # either.
+            try:
+                stdout = run['stdout']
+            except KeyError:
+                stdout = run['Stdout']
+            return device_ctxt['bdevice'] in stdout
```

Comment:
Wouldn't this be better handled in `async_run_on_unit()`?  e.g. paper over the cracks in zaza rather than on every test?